amundsen_application/static/js/config/tests/index.spec.ts

```diff
@@ -33,6 +33,12 @@ describe('feedbackEnabled', () => {
   });
 });
 
+describe('indexUsersEnabled', () => {
+  it('returns whether or not the notifications feature is enabled', () => {
+    expect(ConfigUtils.indexUsersEnabled()).toBe(AppConfig.indexUsers.enabled);
```

Comment:
Thinking about this is bit more, I think an even better thing to do is mock the entire config with variables we define in this file...and verify that `ConfigUtils.getTheThing()` are an exact match with those variables. Thanks for raising the concept I'll file a ticket, these tests really are kind of flimsy as is because they will pass if the values just happen to be the same out of luck.